### PR TITLE
(WIP) Added google analytics to frontend

### PR DESCRIPTION
**Pull request by ywkim312**

### CODE REVIEW

File: templates/frontend/deployment.yaml [hunk 1]
```diff
@@ -26,6 +26,9 @@ spec:
         - name: frontend
           image: "{{ .Values.image.project }}/{{ .Values.frontend.image.repository }}:{{ .Values.frontend.image.tag }}"
           imagePullPolicy: {{ .Values.frontend.image.pullPolicy | default .Values.image.pullPolicy }}
+          env:
```
Comment (longshuicy) [inline templates/frontend/deployment.yaml:29]: Have you tried to add that env to dockerfile here: https://github.com/IN-CORE/incore-ui/blob/develop/Dockerfile#L10. I didn't see it in your WIP PR.

If you have already tried, then maybe the env variable can only be set at build time, not run time.

Comment (ywkim312) [reply templates/frontend/deployment.yaml:29]: I don't think that it needs to be there. I've checked that the docker container already have the env variable and also you can see it in the LENS. Other containers, like data service or others don't have those lines in the Dockerfile but being controlled by helm deployment

Comment (ywkim312) [reply templates/frontend/deployment.yaml:29]: I just tried to add it but it didn't work either.... hmm

Comment (ywkim312) [reply templates/frontend/deployment.yaml:29]: hmmm I searched some, but it is said that it should work with run time setting, but apparently, it doesn't work, so it should be in the build process.

**Comment by ywkim312** [comment]
This will not be needed since the incore-ui's GA_KEY will be embeded in build process not deployment

**Pull request closed without merging**